packages/suite/src/views/password-manager/PasswordManager/PasswordManager.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from '@trezor/components';
+
+import { SectionItem, TextColumn, ActionColumn } from 'src/components/suite';
+import { useDevice, usePasswords } from 'src/hooks/suite';
+import { getNextId } from 'src/utils/suite/passwords';
+
+import { EntryForm } from './EntryForm';
+import { PasswordsList } from './PasswordsList';
+import { TagsList } from './TagsList';
+
+const PasswordManagerBody = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+`;
+
+export const PasswordManager = () => {
+    const {
+        entries,
+        tags,
+        entriesByTag,
+        isSomeTagSelected,
+        config,
+        fileName,
+        selectedTags,
+        setSelectedTags,
+        connect,
+        disconnect,
+        selectedProvider,
+        providerConnecting,
+        savePasswords,
+        device,
+    } = usePasswords();
+
+    const { isLocked } = useDevice();
+    const isDeviceLocked = isLocked();
+
+    const [formActive, setFormActive] = useState<null | number>(null);
+
+    if (!device?.state) {
+        return <SectionItem>Connect and authorized device</SectionItem>;
+    }
+    if (providerConnecting) {
+        return <SectionItem>Connecting...</SectionItem>;
+    }
+
+    if (!selectedProvider || !fileName) {
+        return (
+            <SectionItem>
+                <TextColumn
+                    title="Trezor password manager"
+                    description="Re-implementation of former Trezor Password Manager webextension"
+                />
+                <ActionColumn>
+                    <Button onClick={connect} isDisabled={isDeviceLocked}>
+                        Connect to Dropbox
+                    </Button>
+                    {/* TODO: connect to drive */}
+                </ActionColumn>
+            </SectionItem>
+        );
+    }
+
+    return (
+        <>
+            {/* wrapping into useless divs (at the first glance). however, section item, if not last child adds a border. UI is going to be reworked anyway */}
```

Comment:
It is the `SectionItem` that is useless here, only `display: flex` property is used from it. Please remove it in all cases.

Reply:
Resolved.